docs/intro/quickstart.md

```diff
@@ -0,0 +1,59 @@
+# Quick Start
+
+Get started with Krustlet in three easy steps:
+
+1. Boot a Kubernetes cluster
+2. Boot a Krustlet node
+3. Deploy your first application
+
+As Krustlet is under active development, this guide will help you set up a cluster suitable for evaluation, development,
+and testing purposes.
+
+## Step 1: Boot a Kubernetes Cluster
+
+There are many ways to boot up a Kubernetes cluster. You may choose to get up and running in cloud environments or
+locally on your laptop.
+
+If you have already created a Kubernetes cluster, check out the [system requirements](../topics/system-requirements.md)
+and then proceed to the next step to install Krustlet on your own Kubernetes cluster.
+
+### Cloud-based options
+
+For production use, you'll want to boot a Kubernetes cluster using a fully-managed service provided by a cloud vendor.
+
+- [Azure Kubernetes Service (AKS)](../howto/kubernetes-on-aks.md)
```

Comment:
We should add EKS beneath here with an "under construction" note that links to the issue we have open for it. I want to make sure we are clear that this isn't just for Azure